src/tests/ftest/datamover/dm_dst_create.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/python
+'''
+  (C) Copyright 2020-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+'''
+from data_mover_test_base import DataMoverTestBase
+from os.path import join
+from pydaos.raw import DaosApiError
+import avocado
+
+class DmDstCreate(DataMoverTestBase):
+    # pylint: disable=too-many-ancestors
+    """Data Mover validation for destination container create logic.
+
+    Test Class Description:
+        Tests the following cases:
+            Destination container automatically created.
+            Destination container user attributes match source.
+            Destination container properties match source.
+
+    :avocado: recursive
+    """
+
+    def setUp(self):
+        """Set up each test case."""
+        # Start the servers and agents
+        super().setUp()
+
+        # Get the parameters
+        self.ior_flags = self.params.get(
+            "ior_flags", "/run/ior/*")
+        self.test_file = self.ior_cmd.test_file.value
+        self.cont_type = self.params.get(
+            "cont_type", "/run/dcp/*")
+        self.daos_api = self.params.get(
+            "daos_api", "/run/dcp/*")
+
+        # For dataset_gen and dataset_verify
+        self.obj_list = []
+
+    def run_dm_dst_create(self, tool):
+        """
+        Test Description:
+            Tests Data Mover destination container creation.
+        Use Cases:
+            Create pool1.
+            Create POSIX cont1 in pool1.
+            Create small dataset in cont1.
+            Copy cont1 to a new cont in pool1, with a supplied UUID.
+            Copy cont1 to a new cont in pool1, without a supplied UUID.
+            Create pool2.
+            Copy cont1 to a new cont in pool2, with a supplied UUID.
+            Copy cont1 to a new cont in pool2, without a supplied UUID.
+            For each copy, very container properties and user attributes.
```

Comment:
```suggestion
            For each copy, verify container properties and user attributes.
```

Reply:
Unless there are other needed changes to the PR, I'll leave this spelling correction for a later patch. Thanks!